pkg/compiler/codegen.go

```diff
@@ -988,6 +976,28 @@ func (c *codegen) getLabelOffset(typ labelOffsetType, name string) uint16 {
 	return c.labels[labelWithType{name: name, typ: typ}]
 }
 
+func (c *codegen) emitEquality(expr ast.Expr, op token.Token) {
```

Comment:
I think it can reuse or be merged with `getEqualityOpcode`, they're somewhat similar.

Reply:
Yeah, it was used in switch, could be replaced.